alttpo/LocalGameState.as

```diff
@@ -1293,6 +1298,9 @@ class LocalGameState : GameState {
     for (int i = 0; i < 0x52; i++) {
       r.write_u8(sm_events[i]);
     }
+    
+    r.write_u8(sm_clear);
+    r.write_u8(z3_clear);
```

Comment:
Bump up `const uint8 script_protocol = 0x11;` to `0x12` in `GameState.as` to cover protocol changes. That way players on slightly different versions won't get deserialization errors.